src/_data/sites/es.yml

```
@@ -0,0 +1,570 @@
+#------------------------------------------------------------------------------
+# Spanish Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Luis Morón
+#------------------------------------------------------------------------------
+
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: es
+  flag: 🇪🇸
+  name: Spanish (ES)
+locale: es-ES
+hostname: new.eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: "UA-60915033-1"
```

Let’s leave this blank for now. 
```suggestion
  code: ""
```